dev/devicelab/lib/framework/adb.dart

```diff
@@ -344,6 +364,14 @@ class IosDevice implements Device {
 
   @override
   Future<Null> unlock() async {}
+
+  @override
+  Future<Map<String, dynamic>> getMemoryStats(String packageName) async {
+    return <String, dynamic>{};
```

Comment:
Let's throw instead of returning an empty map. Other methods are stubs because of the way we are using them on iOS. This method we don't want to use at all until we have a real way to get iOS memory stats.